docs/creating_execs_and_libs.rst

```diff
@@ -26,14 +27,17 @@ like ``blank_project`` in the previous section, we can start using BLT's macros.
 Creating an executable is as simple as calling the following macro:
 
 .. literalinclude:: tutorial/calc_pi/CMakeLists.txt
+   :start-after: _blt_tutorial_example_executable_start
```

Comment:
Thanks for fixing these!

Reply:
I actually looked for this feature when i started this and have no idea how i missed it.  So much better than line numbers.